### PR TITLE
Auto scroll

### PR DESCRIPTION
**Pull request by roxanneskelly**
Autoscroll Featured Stories/Announcements in the goto app.

Includes https://github.com/highfidelity/hifi/pull/13806

With fixes for 'pause-on-hover' functionality

### CODE REVIEW

**Comment by roxanneskelly** [comment]
Test Plan.

Follow test plan in https://github.com/highfidelity/hifi/pull/13806

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12108/HighFidelity-Beta-PR13825-5a8da4c.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12108/HighFidelity-Beta-PR13825-5a8da4c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12108/HighFidelity-Beta-Interface-PR13825-5a8da4c.dmg

File: interface/resources/qml/hifi/Feed.qml [hunk 1]
```diff
@@ -134,8 +141,45 @@ Column {
...
             shadowHeight: root.stackedCardShadowHeight;
-            hoverThunk: function () { hovered = true }
-            unhoverThunk: function () { hovered = false }
+            hoverThunk: function () { 
+                hovered = true;
+                autoScrollTimer.stop();
+			}
+            unhoverThunk: function () { 
+                hovered = false;
+                autoScrollTimer.start();
```
Comment (zfox23) [inline interface/resources/qml/hifi/Feed.qml]: We don't want to start the timer unless `root.autoScrollTimerEnabled` is true. Also, we want to restart the timer, not just start it.

Also also, I don't think this works if the user is hovering between cards, right? This is not hugely likely, but I would prefer to not leave gaps.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12108/HighFidelity-Beta-PR13825-5a8da4c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12108/HighFidelity-Beta-Interface-PR13825-5a8da4c.exe

**Comment by zfox23** [comment]
I would definitely like to handle all of those cases!

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12111/HighFidelity-Beta-PR13825-50b0a45.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12111/HighFidelity-Beta-PR13825-50b0a45.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12111/HighFidelity-Beta-Interface-PR13825-50b0a45.dmg

**Comment by rickcarr044** [comment]
Currently testing PR13825 (With Build 9227)

**Comment by rickcarr044** [comment]
Test Results (so far)
Tested on Build 9227

1.  We have not been able to test the scrolling feature as we have been limited to one domain in staging metaverse.  

2.  We noted several possible issues with the Recent Activity list (Click the Featured card's 'red pill').
a. -  There is an image icon above the Recent Activity header - By design or should that be the most recent image in the list?
b.  All of the cards in this list in this build have the same domain name ('hifiqa-master-metaverse-staging").  Previous image look like they are from other domains.  See page 6 where the first and second images look like they are from Dev-Welcome.
c.  Timings (when the snap was taken) appear to be off.  Note in screen capture PR13825.2c that the first image was taken 14 minutes ago, whereas the next three images were taken 1 minute ago.  After posting a new Featured story (card?), the images by RLC044 remained at '1 minute ago', while the image by laseguinolaza had moved to 18 minutes (Previously at 13min).  In two instances, the first blast image was erased from the list.  
pr13825 2c2
pr13825 2b
pr13825 2c
pr13825 2a

**Comment by howard-stearns** [comment]
Re 1: Ops is supposed to be running multiple domains in staging. (Part of the reason they need to, separate from this PR, is that it is currently nearly impossible to run your own named domains in staging. I don't know if they are working on fixing that, or when, but until they do...) Please work with ops to make sure that there are multiple domains so that we can test. This is release-blocker priority.

Re 2 a: The hero picture on a place page (above the "Recent Activity") is the picture specifically uploaded for this purpose by the place owner, or a default image. (If you go to the metaverse web site and log in, the menu at the right has "Places". From their choose a the Edit pencil button for a place you own. On that page, there is a Preview Image "Choose File" button.)

Re 2 b: The content in the staging domains changes from time to time. Sometimes it looks like welcome, sometimes avatar island, etc. I _imagine_ that this is what you are seeing. If we can find that this is definitely wrong, we should make a ticket with repro instructions, but it is not a blocker for the release of this feature.

Re 2 c: The apparently unordered time might be an issue.  While the feeds and /user_stories web pages are supposed to be in "score order", the Recent Activity on user and place pages is supposed to be in descending time order, regardless of score. However, it _might_ be that the order is, say, "create time" order, while the words ("less than a minute ago", "18 minutes ago", "1 minute ago") are in "modification time" order, or something like that. If so, please make a ticket with repro instructions that illustrates the problem, but it is not a blocker for the release of this feature.

**Pull request merged**